app/components/SongCard/tests/index.test.js

```diff
@@ -28,15 +36,81 @@ describe('<SongCard/> tests', () => {
     expect(getAllByTestId('song-card').length).toBe(1);
   });
 
-  it('should play the song when "Play Button" clicked', () => {
+  it('should render No-price translation when trackPrice is not available', () => {
+    const song = {
```

Comment:
maybe you can use the song object defined earlier. change it to let and modify it by using `song = {...song, anynewproperty}`

Reply:
Updated accordigly